datalake/processing_stack.py

```diff
@@ -157,6 +163,95 @@ def __init__(
             application_layer=application_layer,
         ).lambda_invoke
 
+        storage_bucket_arn = aws_ssm.StringParameter.from_string_parameter_attributes(
```

Comment:
I would add `_name` to the variable name to avoid this being confused for the actual ARN value variable.